### PR TITLE
first functional app

### PR DESCRIPTION
**Pull request by joaofranco**
Calling Taxi using Easy Connect

### CODE REVIEW

**Comment by joaofranco** [comment]
i only have read access to this project

File: WatchProject Extension/ETHereManager.swift [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+//  WatchTests
+//
+//  Created by Joao Franco on 13/06/16.
+//  Copyright © 2016 Joao Franco. All rights reserved.
+//
+
+import Foundation
+
+
+class ETHereManager {
```
Comment (zlu) [inline WatchProject Extension/ETHereManager.swift]: what's the point of this file, which is all commented out for the most part?

File: WatchProject Extension/AppleWatchBridge.swift [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+class WatchSessionManager: NSObject, WCSessionDelegate {
+    
+    static let sharedManager = WatchSessionManager()
+    private override init() {
+        super.init()
+    }
+    
+    private let session = WCSession.defaultSession()
+    
+    private var validSession: WCSession? {
```
Comment (zlu) [inline WatchProject Extension/AppleWatchBridge.swift]: misleading naming, seems to be unfinished code.

File: WatchProject Extension/ETHereManager.swift (hunk 1 above)
Comment (joaofranco) [inline WatchProject Extension/ETHereManager.swift]: my bad, i think i removed this file.
I was creating a class to get actual location and near by  locations. To show in the address list

**Pull request closed without merging**